### ISSUE TITLE
What are people using helm for?

### ISSUE THREAD

**Issue by osterman** [cloudposse/docs#311]
## question

* What are people using helm for? I understand it as a concept of charts, etc...

## answer

TL;DR: deployments into kubernetes

`helm` is like `npm` for kubernetes or like a `module` for terraform. You can represent all the resources required for an application as a cohesive unit that is parameterized and templatized so you can install it multiple times. it also comes with a dependency management system, so you can require other charts. It helps to think of kubernetes like a cluster-aware operating system. So instead of installing an application on just one node, you’re installing an application across multiple nodes. For a traditional OS, we’ve always relied on package managers. `helm` solves the equivalent problem but for kubernetes.

A `chart` is just a package. Under the hood, it’s a gzipped tarball that contains some metadata about the application (name, version, dependencies) and then some go templates of kubernetes resources.

**Comment by osterman** [comment]
Here are some examples of charts:
- Maintained by cloudposse: https://github.com/cloudposse/charts/tree/master/incubator
- 100s are maintained by community: https://github.com/helm/charts
